slackbotExercise.py

```diff
@@ -97,7 +99,7 @@ def selectUser(bot, exercise):
         # User should be active and not have done exercise yet
         if user in active_users and not user.hasDoneExercise(exercise):
             bot.user_queue.remove(user)
```

Comment:
Can we still be removing users as we are iterating? Won't this result in an Index Out of Range error now since we're not returning the user out of the loop?